data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -25,6 +25,10 @@ do
     sudo podman inspect "${container}" >& "${ARTIFACTS}/bootstrap/pods/${container}.inspect"
 done
 
+echo "Gathering rendered assets..."
+mkdir -p "${ARTIFACTS}/rendered-assets"
+cp -r /etc/kubernetes/ "${ARTIFACTS}/rendered-assets"
```

Comment:
This feels like it might gather secrets that we don't want gathered by default.  Do you have a list handy of what this is pulling in?

Reply:
> This feels like it might gather secrets that we don't want gathered by default. Do you have a list handy of what this is pulling in?

@sferich888 Do we filter sensitive information before or after we gather?  If before, any advice for @vrutkovs?  This is beyond my bash abilities

Reply:
In vsphere CI we filter out sensitive info in `teardown` container. If all filenames with sensitive data is known it should be removed from `/tmp/artifacts` before `tar cx`

Reply:
> In vsphere CI we filter out sensitive info in `teardown` container. If all filenames with sensitive data is known it should be removed from `/tmp/artifacts` before `tar cx`

I thought so, but let's see what eric says.

Reply:
I think the cloud creds are the only thing the installer asks for that are not generated. 

So..... only scrub that?